test/GithubApi.Users.test.js

```diff
@@ -20,9 +20,62 @@ describe('Given a Github Api URL to do a response time test', () => {
         .set('User-Agent', 'agent');
     });
 
-    it('Then the response time should have been less than five seconds', () => {
+    it('Then the response time should be less than five seconds', () => {
       expect(response.status).to.equal(statusCode.OK);
       expect(rTime).to.satisfy((num) => num < 5000);
     });
+
+    describe('When want to count how many users brings the request by default', () => {
+      let responseCount;
+
+      before(async () => {
+        responseCount = await agent.get(`${urlBase}/users`)
+          .auth('token', process.env.ACCESS_TOKEN)
+          .set('User-Agent', 'agent');
+      });
+
```

Comment:
You dont need to do this again, if you have the response from the previous test use AND

it("Then the response time should be less than 5 secs",()=> { });

it("AND the default amount of users retrieved must've been 30", ()=> { });